### DISCUSSION TITLE
3.15版本报毒

### DISCUSSION THREAD

**Discussion by qinguoluo** [AlistGo/alist#3952] [General]
微软杀毒报毒：Trojan:Win32/Wacatac.B！ml警级：严重类别：特洛伊木马详细信息：这个程序很危险，而且执行来自攻击者的命令。

**Comment by anwen-anyi** [comment]
https://github.com/alist-org/alist/issues/3009#issuecomment-1379690254

**Comment by qinguoluo** [comment]
为什么就这个版本报毒，其他版本就没事呢？有点奇怪。

**Comment by luxiyourbro** [comment]
问题在于这个报毒Windows defender没有“允许”选项
